src/common/seed/images.ts

```diff
@@ -81,18 +81,31 @@ const generateRandomImage = () => {
 	return selectedVariant;
 };
 
-const generateBuild = () => {
+const remainingStatuses = ['building', 'error', 'ready']; //Is it correct way to make global variable in that function?
```

Comment:
why we are need that comment?

What the different where in your app you using constant?

Also you have a statuses const inside your function, why do you use a clone? idk.

Reply:
- never leave comments in code, unless they are crucial. Please, remove this single line comment
- why are they called "remaining"? These are all statuses that we can have for now. It should be called something like "buildStatuses"
- this (each) const, living inside function, will have a function scope. What is the main point of making it in outer space? Please, put it back inside of our function